tests/iohandling/test_parsing.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from unittest.mock import MagicMock, patch
+import sys
+
+from fw_pipeline_io.iohandling import parsing
+
+
+class TestParsing(unittest.TestCase):
+    def setUp(self):
+        # mock a config_file_path
+        self.config_file_path = "tests/test_data/config.yaml"
+        self.tags_file_in_path = "tests/test_data/tags_file_in.yaml"
+
+    def test_create_args_parser_in_algorithm(self):
+        parser = parsing.create_args_parser()
+        args = [self.config_file_path, self.tags_file_in_path]
+
+        args_main = parser.parse_args(args)
+        assert args_main.config_file_path == self.config_file_path
+        assert args_main.tags_file_in_path == self.tags_file_in_path
+
+
+"""
```

Comment:
Is this an example, or just unused code?  If the latter, clearly state or delete when finalizing.

Reply:
removed code